sentry/templates/hooks/clickhouse-init.job.yaml

```diff
@@ -47,4 +48,10 @@ spec:
                 clickhouse-client --database=default --host={{ $clickhouseHost }}-replica-$i.{{ $clickhouseHost }}-replica-headless --port={{ $clickhousePort }} --query="{{ $createQuery }}";
               done
             done
+            {{- else }}
+            for tbl in {{ $tables }}; do
+              clickhouse-client --database=default --host={{ $clickhouseHost }} --port={{ $clickhousePort }} --query="{{ $dropQuery }}";
+              clickhouse-client --database=default --host={{ $clickhouseHost }} --port={{ $clickhousePort }} --query="{{ $createQuery }}";
```

Comment:
External ClickHouse will need a specialised version of `$createQuery` which includes the correct cluster name as the first parameter to Distributed, this should probably be configured via values.

Reply:
I'm not very familiar with ClickHouse, can you help me to check what's correct values can be. I'm running a single node of ClickHouse, not sure is `Distributed` engine is available for a single node or not. But I think this Chart must support both because not everybody is running ClickHouse in clustered mode. If it's ok for you to contact with me with any messenger, I'll able to share my configuration and we can tune this chart to support both env (external single node and cluster).

Reply:
Hey, sorry for taking a while to get back to you on this. I'm on Discord @ `J0sh0nat0r#1337`, feel free to add me or let me know you'd like to using something else :)

Reply:
Discord is ok to me, I have send you a friend request from account `donbeave#8476`.